test/integration/builder/transaction.js

```diff
@@ -395,12 +395,6 @@ module.exports = function(knex) {
     });
 
     it('#2213 - should wait for sibling transactions to finish', function() {
-      if (/redshift/i.test(knex.client.driverName)) {
-        return Promise.resolve();
-      }
```

Comment:
TBH: I'm not really sure why this test case was skipped for `redshift`.  Does anybody have a `redshift` test environment that we can try?

Reply:
maybe redshift not supports savepoints?

Reply:
https://docs.aws.amazon.com/redshift/latest/dg/c_unsupported-postgresql-functions.html
Yes, this is not supported

Reply:
Aiight -- I'll have this fixed in a few mins.  I'm just re-running the unit tests first to make sure I didn't break anything when changing the test cases.